### DISCUSSION TITLE
Hit numbers maximum brightness does not match stable when using Hidden mod

### DISCUSSION THREAD

**Discussion by aeongdesu** [ppy/osu#26112] [Q&A]
### Type

Cosmetic

### Bug description

Instead of the hitnumbers getting brighter in Stable, but Lazer isn't.

The difference is even more noticeable when using Hidden, which makes it harder to read properly. (especially low ar)

tested skins: [lugia](https://cdn.discordapp.com/attachments/1124251914838753281/1184447219831672883/lugiacanreadar11.osk), [Vaxei 1.0](https://drive.google.com/file/d/11LPfXXKhdPbQd-Cwc3uVfeRve-YOIvkn/view?usp=sharing)

### Screenshots or videos

https://github.com/ppy/osu/assets/50764666/85d20b2d-7486-4022-bab6-c4fa1fdf3a68

![](https://github.com/ppy/osu/assets/50764666/e50c5396-1dd0-4a23-85d8-41b79d3f2ef7)
> Screenshot of a frame at maximum brightness on Stable

### Version

2023.1223.0

### Logs

N/A

**Comment by bdach** [comment]
This is an intentional change (https://github.com/ppy/osu/discussions/25745#discussioncomment-7849441).

**Comment by peppy** [comment]
It was intentional, but I'm open to revisiting if push comes to shove. The thing is, the way stable implemented this, it benefitted certain skins more than others (or vice versa, depending on situation) so I'm hoping we can standardise things.
